src/utils/generalUtils.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
+import fetch from 'node-fetch';

```

Comment:
we already have `axios` as dependency. better to reuse existing dependency or Node.js built-in lib?